saskatoon/sitebase/templates/app/detail_views/property/harvest_row.html

```diff
@@ -1,6 +1,18 @@
 {% load i18n %}
 {% load static %}
 <tr>
+    <td>
+        <a href='/harvest/{{harvest.id}}'>{{ harvest.id }}) Harvest by {{ harvest.pick_leader__person__first_name }}</a>
```

Comment:
I find it a bit redundant to have the pickleader's first name to appear twice in this table. Might be relevant to add pickleader's full name or email address in the Pickleader column instead? And eventually a link to the user's profile when such a page will exist?